.github/workflows/ci-sharktank-and-shortfin.yml

```diff
@@ -70,31 +70,5 @@ jobs:
             iree-runtime \
             "numpy<2.0"
 
-      - name: Run llama test
-        run: ./build_tools/integration_tests/llama_export_compile_serve.sh
-
-  test_punet:
-    name: "Integration Tests - punet"
-    runs-on: nodai-amdgpu-mi250-x86-64
-    env:
-      VENV_DIR: ${{ github.workspace }}/.venv
-    steps:
-      - name: "Checkout Code"
-        uses: actions/checkout@v3
-
-      - name: "Setup Python venv"
-        run: python3 -m venv ${VENV_DIR}
-
-      - name: Install pip deps
-        run: |
-          source ${VENV_DIR}/bin/activate
-          python -m pip install --no-compile --upgrade pip
-          pip install --no-compile -r pytorch-rocm-requirements.txt
-          pip install --no-compile -f https://iree.dev/pip-release-links.html --src deps \
-            -e "git+https://github.com/iree-org/iree-turbine.git#egg=iree-turbine"
-          pip install --no-compile -r requirements.txt -r sharktank/requirements-tests.txt -e sharktank/ shortfin/
-
-      - name: Run punet tests
-        run: |
-          source ${VENV_DIR}/bin/activate
-          pytest -v sharktank/ -m model_punet
+      - name: Run shortfin LLM Server Integration Test
+        run: pytest -v build_tools/integration_tests/
```

Comment:
Does the test log anything while it is running? The output is pretty sparse for a test that takes over 3 minutes: https://github.com/nod-ai/SHARK-Platform/actions/runs/11670527836/job/32494907067?pr=373#step:6:1

Could run with `pytest --log-cli-level=info` or some of the other flags: https://docs.pytest.org/en/stable/how-to/usage.html

I'd like to see logs for:
* Artifact downloading (including cache hit/miss)
* Program import (sharktank)
* Program execution (shortfin), from both the client and server

Reply:
Note that pytest will consume log output by default on success. The `--log-cli-level=info` flag will show the captured output.

See also https://docs.pytest.org/en/stable/how-to/logging.html

Reply:
Added logging throughout the run:

```bash
------------------------------------------------------------------------------------------------------------ live log setup -------------------------------------------------------------------------------------------------------------
INFO     conftest:conftest.py:33 Preparing model artifacts...
INFO     conftest:conftest.py:47 Preparing model_path: /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/open-llama-3b-v2-f16.gguf..
INFO     conftest:conftest.py:49 Downloading model SlyEcho/open_llama_3b_v2_gguf open-llama-3b-v2-f16.gguf from Hugging Face...
INFO     conftest:conftest.py:57 Model downloaded to /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/open-llama-3b-v2-f16.gguf
INFO     conftest:conftest.py:63 Preparing tokenizer_path: /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/tokenizer.json...
INFO     conftest:conftest.py:65 Downloading tokenizer openlm-research/open_llama_3b_v2 from Hugging Face...
INFO     conftest:conftest.py:70 Tokenizer saved to /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/tokenizer.json
INFO     conftest:conftest.py:78 Exporting model with following settings:
  MLIR Path: /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/model.mlir
  Config Path: /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/config.json
  Batch Sizes: 1,4
INFO     conftest:conftest.py:96 Model successfully exported to /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/model.mlir
INFO     conftest:conftest.py:100 Compiling model to /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/model.vmfb
INFO     conftest:conftest.py:111 Model successfully compiled to /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/model.vmfb
INFO     conftest:conftest.py:126 Saving edited config to: /tmp/pytest-of-amd/pytest-53/cpu_llm_server_test0/edited_config.json

INFO     conftest:conftest.py:127 Config: {
  "module_name": "module",
  "module_abi_version": 1,
  "max_seq_len": 2048,
  "attn_head_count": 32,
  "attn_head_dim": 100,
  "prefill_batch_sizes": [
    1,
    4
  ],
  "decode_batch_sizes": [
    1,
    4
  ],
  "transformer_block_count": 26,
  "paged_kv_cache": {
    "block_seq_stride": 16,
    "device_block_count": 256
  }
}
INFO     conftest:conftest.py:130 Model artifacts setup successfully
INFO     conftest:conftest.py:140 Finding available port in range 8000-8100...
INFO     conftest:conftest.py:149 Found available port: 8000
INFO     conftest:conftest.py:184 Starting LLM server...
INFO     conftest:conftest.py:158 Waiting for server to start at http://localhost:8000...
INFO     conftest:conftest.py:163 Server successfully started
------------------------------------------------------------------------------------------------------------- live log call -------------------------------------------------------------------------------------------------------------
INFO     cpu_llm_server_test:cpu_llm_server_test.py:33 Generating request...
INFO     cpu_llm_server_test:cpu_llm_server_test.py:46 Prompt text:
INFO     cpu_llm_server_test:cpu_llm_server_test.py:47 1 2 3 4 5 
INFO     cpu_llm_server_test:cpu_llm_server_test.py:50 Generate endpoint status code: 200
INFO     cpu_llm_server_test:cpu_llm_server_test.py:52 Generated text:
INFO     cpu_llm_server_test:cpu_llm_server_test.py:84 6 7 8 8 8 8 8 8 
```